diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/WhiteSpaceRule.kt

```diff
@@ -111,13 +115,13 @@ class WhiteSpaceRule(private val configRules: List<RulesConfig>) : Rule("horizon
             CONSTRUCTOR_KEYWORD -> handleConstructor(node)
             in keywordsWithSpaceAfter -> handleKeywordWithParOrBrace(node)
             // operators and operator-like symbols
-            OPERATION_REFERENCE, COLONCOLON, DOT, ARROW, SAFE_ACCESS -> handleBinaryOperator(node)
+            OPERATION_REFERENCE, COLONCOLON, DOT, ARROW, SAFE_ACCESS, EQ -> handleBinaryOperator(node)
```

Comment:
`EQ` should be surrounded by spaces no only in properties but everywhere else (assignments, named function arguments etc)

Reply:
But the logic is the same. I add test for this case.